Cargo.toml

```diff
@@ -110,3 +110,6 @@ path = "examples/ureq/search.rs"
 name = "seek_track"
 required-features = ["env-file", "cli", "client-ureq"]
 path = "examples/ureq/seek_track.rs"
+
+[profile.dev]
+split-debuginfo = "unpacked"
```

Comment:
is this to get faster compilation times on MacOS? It'll become the default in the  next version (I think) https://github.com/rust-lang/cargo/pull/9298 so you should add a note so that we can remove it in the future

Reply:
Removed :)